pkg/controller/dataimportcron-controller.go

```diff
@@ -394,10 +394,11 @@ func (r *DataImportCronReconciler) update(ctx context.Context, dataImportCron *c
 		return res, err
 	}
 
-	// We use the poller returned reconcile.Result for RequeueAfter if needed
-	// skip if we disabled schedule
+	// Skip if schedule is disabled
 	if isImageStreamSource(dataImportCron) && dataImportCron.Spec.Schedule != "" {
-		res, err := r.pollImageStreamDigest(ctx, dataImportCron)
+		// We use the poll returned reconcile.Result for RequeueAfter if needed
+		// res is the func returned result, so it must not be block-scoped
+		res, err = r.pollImageStreamDigest(ctx, dataImportCron)
```

Comment:
My concern is that this is so delicate we may step on it again in the future,
maybe we should decide whether requeue happens more explicitly?

BTW the reasoning for this was to get a uniform way of assigning errors in this module (= to :=) IIRC
instead of mixing them up

Reply:
Rewrited the code a bit. I think it's enough.